app/views/static_pages/landing_page.html.erb

```diff
@@ -1,8 +1,91 @@
+        Start now
+        <svg class="govuk-button__start-icon" xmlns="http://www.w3.org/2000/svg" width="17.5" height="19" viewBox="0 0 33 40" aria-hidden="true" focusable="false">
+          <path fill="currentColor" d="M0 0h13l20 20-20 20H0l20-20z"/>
+        </svg>
+      </a>
+    </p>
+
+    <h2 class="govuk-heading-m govuk-!-margin-top-4">Further information</h2>
+
+    <p class="govuk-body">
+      You must apply between September <%= @academic_year.start_year.to_s %> and January <%= @academic_year.end_year.to_s %>.
+    </p>
+
+    <p class="govuk-body">
+      If you are eligible for an additional payment, we will collect personal information to verify your claim and process your payment.
+    </p>
+
+    <p class="govuk-body">
+      Payments are given up to 16 weeks from applying. You will receive an approval email with further information on your application.
+    </p>
+
+    <details class="govuk-details" data-module="govuk-details">
+      <summary class="govuk-details__summary">
+        <span class="govuk-details__summary-text">The different additional payments</span>
+      </summary>
+      <div class="govuk-details__text">
+        <p class="govuk-body">
+          There are 2 types of additional payment. You may be eligible for both payments, but you will only be able to apply for one each year.
+        </p>
+
+        <ul class='govuk-list govuk-list--bullet'>
+          <li>
+            <a href="https://www.gov.uk/guidance/early-career-payments-guidance-for-teachers-and-schools" class="govuk-link govuk-link--no-visited-state" target="_blank">Early-career payment (opens in new tab)</a> - payment of up to £7,500 for eligible teachers including language teachers.
+          </li>
+          <li>
+            <a href="https://www.gov.uk/guidance/levelling-up-premium-payments-for-teachers" class="govuk-link govuk-link--no-visited-state" target="_blank">Levelling up premium payment (opens in new tab)</a> - payment of up to £3,000 for eligible teachers in certain schools, including computing teachers.
+          </li>
+        </ul>
+
+        <p class="govuk-body">
+          Both schemes offer the largest payments to those teaching in more disadvantaged areas and schools.
+        </p>
+      </div>
+    </details>
+
+    <h3 class="govuk-heading-s govuk-!-margin-top-6">Claim back your student loan repayments</h3>
+
+    <p class="govuk-body">
+      If you have a student loan, you may be able to <a href="https://www.gov.uk/guidance/teachers-claim-back-your-student-loan-repayments" class="govuk-link govuk-link--no-visited-state" target="_blank">claim back student loan repayments (opens in new tab)</a> made while employed as a teacher. This is separate and you can apply for this whether or not you receive an early-career payment or levelling up premium payment.
```

Comment:
I'm not sure if the designers know or care about the but GOV.UK provide a component for this which has subtly different behaviour (and makes the colour the link goes slightly differently to the `<a>` element default)